-------------------------------------------------------------------------------
Test set: test.test.rrrTest
-------------------------------------------------------------------------------
Tests run: 6, Failures: 2, Errors: 0, Skipped: 0, Time elapsed: 19.598 sec <<< FAILURE!
main5(test.test.rrrTest)  Time elapsed: 0.009 sec  <<< FAILURE!
java.lang.AssertionError: This is forced error expected [false] but found [true]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:286)
	at test.test.rrrTest.main5(rrrTest.java:34)

main8(test.test.rrrTest)  Time elapsed: 0.002 sec  <<< FAILURE!
java.lang.AssertionError: This is forced error expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:286)
	at test.test.rrrTest.main8(rrrTest.java:49)

